sos/policies/distros/__init__.py

```diff
@@ -354,6 +385,54 @@ def _configure_low_priority(self):
         except Exception as err:
             self.soslog.error(f"Error setting report niceness to 19: {err}")
 
+    def prompt_for_upload_s3_access_key(self):
+        """Should be overridden by policies to determine if an access key needs
+        to be provided for upload or not
+        """
+        if not self.get_upload_s3_access_key():
+
+            msg = ("Please provide the upload access key for bucket %s via "
+                   "endpoint %s: " % (self.get_upload_s3_bucket(),
+                                      self.get_upload_s3_endpoint()))
```

Comment:
Please convert to f-strings, we're trying to make f-strings the standard in the project.

Reply:
Cool, I prefer them anyway, but was maintaining any existing styles I observed from other plugins since I forgot which contribution page I thought I saw a suggestion of moving to f-strings.